### DISCUSSION TITLE
No Recommended hides the blog carousel atop "Blog Subscriptions"

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#338] [Lounge]
(pro tip: select your discussion category before typing out your message. plz upvote [this](https://github.com/github/feedback/discussions/7934), tyvm :P aaaanyways...)

https://www.tumblr.com/timeline/blog_subscriptions contains a blog recommendation carousel that differs somewhat from those on the dashboard. It only contains blogs you're subscribed to.

That I know of, this is the only place in the Tumblr UI where this information is listed(?) - I imagine that, if the Blog Subscriptions page were to be worked on by Staff until it were deemed fit to link on the website proper, there would be more UI here.

In any case, No Recommended hides this carousel. I am honestly undecided about whether this makes sense or not; on the one hand, it is most definitely a recommended blog carousel; on the other hand, it sort of feels like it's being used as placeholder UI.

**Comment by marcustyphoon** [comment]
Oh, I just noticed that https://www.tumblr.com/timeline/crushes, another "secret dashboard" shared by Cyle, quite explicitly uses the blue header atop its carousel to explain what the page is for, so it being hidden can be confusing. Based on this, I think I will think of this behavior as a bug, and work on a PR to only enable No Recommended on the dashboard.

(On the other hand, though, I'm sure regular users of those pages might not want a carousel at the top of them all the time... hmmmmm)

**Comment by AprilSylph** [comment]
The option *is* called "Hide blog carousels between posts", not "Hide all blog carousels". This might be fixed easily with a simple check for if there are preceding posts (`[data-id] ~ whatever`).

**Reply by marcustyphoon** [reply]
Good point. And hm, that's probably easier than what I was thinking (`document.querySelector('[data-timeline=/v2/timeline/dashboard]').querySelectorAll(...)`), as long as you can't ever have the top of the dash be, say, an ad and then a carousel, which... I imagine you probably can't?

**Reply by AprilSylph** [reply]
Yeah no, there are blog carousels on `/tagged/` pages too which the option should definitely hide. And if a tag or someone's dash is so empty that there is a blog carousel before any posts... the carousel probably isn't an issue, and the feature would still be behaving as described.

**Reply by marcustyphoon** [reply]
Oh right. Yep, that's true - evidently "between posts" is much more precise than I realized!
